pkg/kubelet/kuberuntime/kuberuntime_container_linux.go

```diff
@@ -89,6 +90,21 @@ func (m *kubeGenericRuntimeManager) generateLinuxContainerConfig(container *v1.C
 
 	lc.Resources.HugepageLimits = GetHugepageLimitsFromResources(container.Resources)
 
+	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.NodeSwapEnabled) {
```

Comment:
I think we need a similar change in `ResourceConfigForPod` for pod level cgroup settings created by pod cgroup manager.  I would expect them to match the container settings.  I _think_ memory backed volumes could ultimately use swap, but would like @sjenning to confirm.  Either way, the cgroup settings for memory should match pod and container scopes.